pkgs/applications/version-management/git-stack/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
```

```suggestion
, fetchFromGitHub
, rustPlatform
```

Fetchers come first.